NickvisionMoney.Shared/Helpers/CurrencyHelpers.cs

```diff
@@ -39,6 +39,19 @@ public static string ToAmountString(this decimal amount, CultureInfo culture, bo
             };
             result = string.Format(formatString, result);
         }
+        if("0" != CultureInfo.CurrentCulture.NumberFormat.NativeDigits[0])
+        {
+            result = result.Replace("0", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[0])
+                           .Replace("1", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[1])
+                           .Replace("2", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[2])
+                           .Replace("3", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[3])
+                           .Replace("4", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[4])
+                           .Replace("5", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[5])
+                           .Replace("6", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[6])
+                           .Replace("7", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[7])
+                           .Replace("8", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[8])
+                           .Replace("9", CultureInfo.CurrentCulture.NumberFormat.NativeDigits[9]);
+        }
```

Comment:
This will cause validation of amount values in dialogs to fail, because NativeDigits are not digits for dotnet (see the link I attached in the issue).
Also I think it would be good to have a setting to disable native digits.
I can work on this a bit later.

Reply:
I made it so that input dialogs will use Latin digits and only labels will show native digits. Because it'd be impossible to input and parse non-ASCII native digits according to that comment. 

https://imgur.com/a/hfUGpSe
*I don't have the font to display eastern Arabic digits but it's there*

> Also I think it would be good to have a setting to disable native digits.

Maybe another field in custom currency: `Display Native Digits`

Reply:
Requested a review to trigger CI :)

Reply:
> Maybe another field in custom currency: Display Native Digits

I think it should be global, not per-account. If a user don't want to use native digits they probably don't want to use it everywhere.

> Because it'd be impossible to input and parse non-ASCII native digits according to that comment.

We can create a helper function to convert native digits in string back to latin and then parse to double.

Reply:
Global works :)